rdtools/plotting.py

```diff
@@ -256,14 +259,61 @@ def soiling_rate_histogram(soiling_info, bins=None):
 
     soiling_summary = soiling_info['soiling_interval_summary']
     fig, ax = plt.subplots()
-    ax.hist(100.0 * soiling_summary.loc[soiling_summary['valid'], 'soiling_rate'],
-            bins=bins)
+    ax.hist(100.0 * soiling_summary.loc[soiling_summary['valid'],
+                                        'soiling_rate'], bins=bins)
     ax.set_xlabel('Soiling rate (%/day)')
     ax.set_ylabel('Count')
 
     return fig
 
 
+def tune_clip_filter_plot(power_ac, clipping_mask, display_web_browser=True):
```

Comment:
I am ambivalent about the default value for `display_web_browser` if we think most users will use it in a notebook, I think `False` is preferable. But if people will use it in spyder etc then maybe `True` is better.